src/EnergyPlus/HighTempRadiantSystem.cc

```diff
@@ -440,7 +440,14 @@ namespace HighTempRadiantSystem {
             }
 
             // Process the temperature control type
-            highTempRadSys.ControlType = static_cast<RadControlType>(getEnumValue(radControlTypeNamesUC, state.dataIPShortCut->cAlphaArgs(6)));
+            if (state.dataIPShortCut->lAlphaFieldBlanks(6)) {
+                ShowWarningEmptyField(state, eoh, state.dataIPShortCut->cAlphaFieldNames(6), "OperativeTemperature");
+                highTempRadSys.ControlType = RadControlType::OperativeControl;
```

Comment:
A6 will never be blank since it has a default. Old line 443 is all that is needed.

    A6 , \field Temperature Control Type
        \note Temperature type used to control unit
        \type choice
        \key MeanAirTemperature
        \key MeanRadiantTemperature
        \key OperativeTemperature
        \key MeanAirTemperatureSetpoint
        \key MeanRadiantTemperatureSetpoint
        \key OperativeTemperatureSetpoint
        \default OperativeTemperature